Windows: Module not found: Can't resolve, the path.

Description:
I encountered a strange issue this morning(11.1.2021), when trying to do an `npm start` on my windows machine.
The error on the console says:

```
./node_modules/@scandipwa/scandipwa/src/index.js
Module not found: Can't resolve 'C:UsersxxxDesktopscandipwa-app	est-app
```
i am trying to create a scandipwa app using` npx create-scadipwa-app test-app` in my C:/Users/xxx/Desktop/scandipwa-app folder.
It seems it is ignoring the / (slash) in the folder path.

Expected behavior:
to load scandipwa theme on localhost:3000/

Screenshots, Video, Logs

To Reproduce:

Open terminal
npx create-scandipwa-app
npm install.
Versions:

node -v: v12.13.1
npm -v: 6.12.1
OS: Window

--- a/build-packages/scandipwa-extensibility/build-config/webpack-extension-import-helper-loader/index.js
+++ b/build-packages/scandipwa-extensibility/build-config/webpack-extension-import-helper-loader/index.js
@@ -6,7 +6,11 @@ const path = require('path');
  * But it will occur before ReactDOM.render()
  */
 module.exports = function injectImports(source) {
-    const injectablePath = path.resolve(__dirname, '..', '..', 'runtime-helpers');
+    const injectablePath = path
+        .resolve(__dirname, '..', '..', 'runtime-helpers')
+        .split(path.sep)
+        .join(path.posix.sep);
+
     const injectableCode = `import '${injectablePath}';\n`;
 
     const importMatcher = /^import .+$/gm;

--- a/build-packages/scandipwa-extensibility/build-config/webpack-extension-import-loader/index.js
+++ b/build-packages/scandipwa-extensibility/build-config/webpack-extension-import-loader/index.js
@@ -48,7 +48,10 @@ const getExtensionImports = (pathname) => {
         return [];
     }
 
-    return findPluginFiles(pathname).map((pluginFile) => `require('${pluginFile}').default`);
+    return findPluginFiles(pathname).map((pluginFile) => {
+        const pluginFilePath = pluginFile.split(path.sep).join(path.posix.sep);
+        return `require('${pluginFilePath}').default`;
+    });
 };
 
 module.exports = function injectImports(source) {

--- a/build-packages/scandipwa-scripts/lib/alias.js
+++ b/build-packages/scandipwa-scripts/lib/alias.js
@@ -62,9 +62,11 @@ const preferenceAliases = extensions.reduce((acc, extension) => {
         return acc;
     }
 
+    const posixPackagePath = packagePath.split(path.sep).join(path.posix.sep);
+
     return {
         ...acc,
-        [`${preference}/*`]: [path.relative(process.cwd(), packagePath)]
+        [`${preference}/*`]: [path.relative(process.cwd(), posixPackagePath)]
     };
 }, {});
 
@@ -78,9 +80,10 @@ const { parentThemeAliases } = Object.entries(aliasMap).reduceRight(
     (acc, [, aliasPathMap]) => {
         Object.entries(aliasPathMap).forEach(
             ([alias, pathname], i) => {
+                const posixPathname = pathname.split(path.sep).join(path.posix.sep);
                 acc.aliasStack[i].push(
                     // it is required to be relative, otherwise it does not work
-                    `${path.relative(process.cwd(), pathname)}/*`
+                    `${path.relative(process.cwd(), posixPathname)}/*`
                 );
 
                 acc.parentThemeAliases[`${ alias }/*`] = Array.from(acc.aliasStack[i]);
@@ -98,7 +101,7 @@ const { parentThemeAliases } = Object.entries(aliasMap).reduceRight(
 // TODO: generate jsconfig.js for VSCode suggestions
 const jsConfig = {
     compilerOptions: {
-        baseUrl: `.${path.sep}`,
+        baseUrl: `.${path.posix.sep}`,
         jsx: 'react',
         paths: {
             ...parentThemeAliases,

--- a/build-packages/webpack-i18n-plugin/index.js
+++ b/build-packages/webpack-i18n-plugin/index.js
@@ -23,7 +23,11 @@ const addParsedVariableToModule = (parser, name) => {
         return false;
     }
 
-    const pathToTranslationFunction = path.join(__dirname, 'lib/translation-function.js');
+    const pathToTranslationFunction = path
+        .join(__dirname, 'lib/translation-function.js')
+        .split(path.pos)
+        .join(path.posix.pos);
+
     const expression = `require(${ JSON.stringify(pathToTranslationFunction) })`;
     const deps = [];
 

--- a/build-packages/webpack-i18n-runtime/build-config/webpack-i18n-import-loader/index.js
+++ b/build-packages/webpack-i18n-runtime/build-config/webpack-i18n-import-loader/index.js
@@ -28,8 +28,9 @@ function getFilePathsForLocale(locale) {
     );
 }
 
-function generateImportDeclaration(path, chunkName) {
-    return `import(/* webpackMode: "lazy", webpackChunkName: "${chunkName}" */ '${path}')`;
+function generateImportDeclaration(pathname, chunkName) {
+    const posixPathname = pathname.split(path.sep).join(path.posix.sep);
+    return `import(/* webpackMode: "lazy", webpackChunkName: "${chunkName}" */ '${posixPathname}')`;
 }
 
 function generateLocaleMapContents(localePathMap, defaultLocale) {

